src/server/getters/get-user-trading-positions.ts

```diff
@@ -169,3 +187,51 @@ async function getEthEscrowedInValidityBonds(db: Knex, marketCreator: Address):
   }
   return totalValidityBonds;
```

Comment:
It looks like this method returns Promise<BigNumber> but ZERO is defined as new BigNumber(0) which isn't a promise. Also, I am unsure if marketRow has data where totalValidityBonds turns into a promise? Maybe I'm missing something.

Reply:
Async functions have syntax sugar to wrap returned values in `Promise.resolve`.

From recent PR

```
  return new Promise<PlatformActivityResult>((resolve, reject) => {
    resolve(result);
  });
```

In any function (including non-async) this could instead be:

```
return Promise.resolve(result);
```

Or in an async function:

```
return result; // automagically wrapped in Promise.resolve()
```